Softeq.XToolkit.Common.iOS/Extensions/NSStringExtensions.cs

```diff
@@ -79,6 +79,23 @@ public static NSMutableAttributedString BuildAttributedString(this string input)
             return self;
         }
 
+        public static NSMutableAttributedString LinksFont(
+            this NSMutableAttributedString self, UIFont font)
+        {
+            self.EnumerateAttributes(
+                new NSRange(0, self.Length),
+                NSAttributedStringEnumeration.Reverse,
```

Comment:
Why is reverse used here? Does it make any difference?

Reply:
Will check this. The code was got from here: https://stackoverflow.com/questions/35053694/uitextview-linktextattributes-font-attribute-not-applied-to-nsattributedstring

Reply:
I didn't find any difference, changed to .None